### PR TITLE
test(cypress): Adds image comparisons to verify features loaded

### PR DESCRIPTION
**Pull request by justin-bits**
This adds image comparisons to existing tests.

Merge https://github.com/ngageoint/opensphere/pull/543 first, or just close that and merge this.

### CODE REVIEW

**Comment by justin-bits** [comment]
Investigating failures.

File: .travis.yml [hunk 1]
```diff
@@ -24,20 +24,17 @@ install:
...
 - chmod ugo+x ./deploy.sh
 - "./deploy.sh"
+after_failure:
+- "./cypress/support/s3-artifact-upload.sh"
 branches:
   except:
   - "/^v\\d+\\.\\d+\\.\\d+$/"
 env:
   global:
-  - secure: LdQr1FTRbiLboL5DQsJ31ngiIqJDC8gAwKzUVCzv3G/tcLFokwmGJUB5pEpvSgmbWVhNEqqqNwymj+7ib2ylx4QiNVDFWqRSZ+if9mFoCiTmvrJeKxmxOs0aPZtsb5sYHQqf5XLj59275PH43PO9zpJSfiOdekm924pEOqUa24vI4LnGRMDcgmWd77QURbLCOIOXkfWH15TahvTqb7Z8C5pNB5Qt5jlsd8/iPCFriZrCcEFaMtd3psKjsTsQEpyG3kAHAsbQbbK8hc1tF7mol9sypvKRWpUvrpR4JUBNSLnLgposzHF1LD4ycKoqI0/o1iogo7CQbxJ/QFwbzE2DbM4T9J6A5mt426ydT3nByBt04nlcnIgvv0OXZmlmT/Aqhk6xBtNm9c676oqJ6+q3seQoNpgnFxeCWyi7rtJnQQTglWoMTxCoPpE2gipvZ+U/IfUYi5pxz78AcvVHAGPVTto3mwYuzPHEBeq9ecbKDtaPnMOSh96/z65LGhXrn2N9vTVRqnYq4h2cpgPwRGLn2S6HX9o8Lq3MuHU/PUg7gShnjKx0rjC94/g/5IXX0mzOboNEig4OrLK/J2Z+X3T+J2o61dKMtF3ixXDDMWZGeT+KiEfMxP0ivuaIYXCG2ukCI3XYt0Kn8NLQI/6yKUVWsB8DMPkbZVvrfeoxZ9N7Rh8=
```
Comment (schmidtk) [inline .travis.yml]: Two of these are for Surge.sh deployments. @wallw-bits could you please add those to the Travis settings before we merge this? The other is `GITHUB_API_TOKEN` which we replaced with `GH_TOKEN` in the Travis settings, unless I'm mistaken and those aren't the same thing.

**Comment by justin-bits** [comment]
@schmidtk, file removed, gitignore updated. This should be good now.

**Comment by justin-bits** [comment]
- Snapshots now referred to as comparisons
- Thresholds/delay used in individual tests moved / cleaner tests
- A few images updated
- Pixel level threshold used to remove some of the rendering noise between Electron/Chrome

**Pull request merged**